openapi/endpoints/funding-channel/funding-channels-get.yaml

```
@@ -0,0 +1,26 @@
+get:
+  tags:
+    - Funding Channel
+  summary: List funding channels
+  description: Retrieves a list of fundingChannels for the given account
```

```suggestion
  description: Retrieves a list of Funding Channels for the given account
```

Can we do this everywhere